packages/component-base/src/uri-utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Check if a path matches to any within an array of paths
+ *
+ * @param {string} pathToMatch
+ * @param {Array<String>} paths
+ */
+export function isMatchingPath(pathToMatch, paths) {
+  if (!paths || (!pathToMatch && pathToMatch !== '')) {
```

Comment:
Usually it's responsibility of the caller to make sure undefined paths aren't passed given that the helper only accepts strings. It seems `__pathExists` already checks that before calling this helper, so maybe we don't need these checks.

Reply:
Removed the checks.